backend/tracim_backend/lib/utils/authorization.py

```diff
@@ -425,14 +425,9 @@ def check(self, tracim_context: TracimContext) -> bool:
 is_todo_owner = TodoOwnerChecker()
 is_assignee = TodoAssigneeChecker()
 can_edit_todo = OrAuthorizationChecker(
-    AndAuthorizationChecker(is_contributor, is_todo_owner),
-    is_assignee,
-    is_content_manager,
-    is_workspace_manager,
-)
-can_delete_todo = OrAuthorizationChecker(
-    AndAuthorizationChecker(is_contributor, is_todo_owner), is_content_manager, is_workspace_manager
+    is_assignee, is_todo_owner, is_content_manager, is_workspace_manager,
```

Comment:
This mean reader can edit todo ? Are we sure we want this ?
Can you update doc/roles.md with the todo rules ?